Makefile

```diff
@@ -1,16 +1,16 @@
 all:
 
 debug:
-	@node_modules/.bin/node-debug fathom.js
+	@npm run debug
 
 lint:
-	@node_modules/.bin/jshint fathom.js
+	@npm run lint
 
 test:
-	@node_modules/.bin/mocha
+	@npm run test
 
 debugtest:
-	@node_modules/.bin/mocha -d
+	@npm run test -- -d
```

Comment:
Probably not at the moment. There might have been some in there at the beginning of the project, or it could just be me being new to the ecosystem; I wasn't aware of npm scripts.